### ISSUE TITLE
Error 500 when trying to remove automatically posted timers

### ISSUE THREAD

**Issue by ppfeufer** [pvyParts/allianceauth-signal-pings#1]
This extention - https://gitlab.com/ErikKalkoken/aa-structures - can post timers for structires autuomatically to the timerboard and as far as I understand the "creator" of such a timer is optional.

Now `signalpings` seems to assume the creator is mandatory, which prevents from being able to remove these timers and raises an Error 500. Also the üing on creation is not sent.

```
[2020-07-14 00:55:03,017: INFO/MainProcess] [Structures] TN-NT: Received 1 new notifications from ESI
[2020-07-14 00:55:03,043: ERROR/MainProcess] [Structures] 1263919265: Failed to add timer from notification: 'NoneType' object has no attribute 'profile'
Traceback (most recent call last):
File "/home/tnnt_auth/auth-system/venv/auth/lib/python3.7/site-packages/structures/models/notifications.py", line 1169, in process_for_timerboard
timer.save()
File "/home/tnnt_auth/auth-system/venv/auth/lib/python3.7/site-packages/django/db/models/base.py", line 741, in save
force_update=force_update, update_fields=update_fields)
File "/home/tnnt_auth/auth-system/venv/auth/lib/python3.7/site-packages/django/db/models/base.py", line 790, in save_base
update_fields=update_fields, raw=raw, using=using,
File "/home/tnnt_auth/auth-system/venv/auth/lib/python3.7/site-packages/django/dispatch/dispatcher.py", line 175, in send
for receiver in self._live_receivers(sender)
File "/home/tnnt_auth/auth-system/venv/auth/lib/python3.7/site-packages/django/dispatch/dispatcher.py", line 175, in <listcomp>
for receiver in self._live_receivers(sender)
File "/home/tnnt_auth/auth-system/venv/auth/lib/python3.7/site-packages/signalpings/signals.py", line 53, in timer_saved
logger.debug("New signal for %s" % instance.user.profile.main_character, flush=True)
AttributeError: 'NoneType' object has no attribute 'profile'
[2020-07-14 00:55:03,064: ERROR/MainProcess] [Structures] 1263927978: Failed to add timer from notification: 'NoneType' object has no attribute 'profile'
Traceback (most recent call last):
File "/home/tnnt_auth/auth-system/venv/auth/lib/python3.7/site-packages/structures/models/notifications.py", line 1169, in process_for_timerboard
timer.save()
File "/home/tnnt_auth/auth-system/venv/auth/lib/python3.7/site-packages/django/db/models/base.py", line 741, in save
force_update=force_update, update_fields=update_fields)
File "/home/tnnt_auth/auth-system/venv/auth/lib/python3.7/site-packages/django/db/models/base.py", line 790, in save_base
update_fields=update_fields, raw=raw, using=using,
File "/home/tnnt_auth/auth-system/venv/auth/lib/python3.7/site-packages/django/dispatch/dispatcher.py", line 175, in send
for receiver in self._live_receivers(sender)
File "/home/tnnt_auth/auth-system/venv/auth/lib/python3.7/site-packages/django/dispatch/dispatcher.py", line 175, in <listcomp>
for receiver in self._live_receivers(sender)
File "/home/tnnt_auth/auth-system/venv/auth/lib/python3.7/site-packages/signalpings/signals.py", line 53, in timer_saved
logger.debug("New signal for %s" % instance.user.profile.main_character, flush=True)
AttributeError: 'NoneType' object has no attribute 'profile'
```

See also here - https://gitlab.com/ErikKalkoken/aa-structures/-/issues/27

**Comment by pvyParts** [comment]
Sure I can fix that. Didn't have any testing in place for no creator, as It couldn't happen from within auth.

**Comment by pvyParts** [comment]
fixed with version 0.0.5

500's wont happen anymore. However i am not sure how i want to handle those type of "scenarios" (Missing data that can only happen  through 3rd party input )given that atm this already has probably pinged in discord, it will just not post them to the webhook for now. I'll leave this open till i decide what way to go.

Happy to hear your input.

**Comment by ppfeufer** [comment]
Have to wait until we get the next thing reinforced now :-)

But I would say, since the Creator is optional,don't rely on it and only use it if it's not NULL.
